relayer/src/lib.rs

```diff
@@ -73,6 +74,22 @@ struct WebState {
     contract: CAPE<EthMiddleware>,
     nonce_count_rule: NonceCountRule,
     gas_limit: u64,
+    block_submission_mutex: Arc<Mutex<u64>>,
```

Comment:
`Arc<Mutex<()>>` is preferred if we're not actually using the value in the mutex.